### DISCUSSION TITLE
How can I contribute translations?

### DISCUSSION THREAD

**Discussion by salihyalcin** [webrtc-for-the-curious/webrtc-for-the-curious#35] [General] [answered]
Hello, thanks for this kind of information. Will you consider to translate this repo to another languages. Why I am asking this kind of question is I would like to translate these informations to the my native language (Turkish) if you mind ? 

Thanks at all.

**Comment by Sean-Der** [comment]
@salihyalcin that would be amazing! Thank you so much for getting involved :)

Let me move the repo around so we have dedicated folders per language.

**Comment by Sean-Der** [comment, marked as answer]
Done! We now have https://github.com/webrtc-for-the-curious/webrtc-for-the-curious/tree/master/content.tr which directly mirrors english.

If anyone else is interested in contributing translations it would be really appreciated :)

**Comment by salihyalcin** [comment]
Thanks at all, my main reason behind this translation is learning WebRTC and let people learn from from their native language. Thanks for contributing. I will do my best to translate.
